src/main/java/com/google/step/finscholar/data/CollegeData.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/** Holder for hardcoded College objects. */
+public final class CollegeData {
+  public CollegeData () {}
```

Comment:
(I'm not sure about this comment. Correct me if I'm  wrong.)
Do you need this constructor? This class doesn't have any instance variables or instance methods.

Reply:
Oh yeah. In JS, the default constructor has to be declared explicitly, I guess that's why I did that. But you're right, I don't need this since it's java. I got the two confused lol.